.github/workflows/web.yml

```diff
@@ -1,4 +1,6 @@
 name: web
+run-name: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch') && format('web nightly {0}', github.ref_name) }}
```

Comment:
where is the usage of input `is_nightly` ? :(

Reply:
![image](https://github.com/centreon/centreon/assets/144020537/e29760b1-09fd-40ca-a3fc-7d69b38f177e)

with the actual update we have this output , but while adding && inputs.is_nightly == 'true' we will get false is it necessary to adding it?

Reply:
this is because you need to add `|| ''` at the end of the condition

Reply:
check here : https://github.com/centreon/centreon/pull/2872/files#diff-5d8adc0ecc6d4ffce45387b7523fb6cca8539efedfeff49cac7a722a10c8160cR2